examples/kafka-hub/A Guide on implementing Websub Hub backed by Kafka Message Broker.md

```diff
@@ -198,6 +204,12 @@ Then execute the below command to run the program.
 BAL_CONFIG_FILES=/path/to/Config.toml bal run target/bin/wbsbpublisher.jar
 ```
 
+Running the publisher on docker.
+
+```sh
+docker run --rm --network host --name websub-publisher -e TOPIC_NAME="<topic-name>" ayeshalmeida/wbsbpublisher:4.0.0
```

Comment:
We are using different versions of the `wbsbpublisher` docker image for running publisher and subscriber. Can't we use the same?

Reply:
Subscriber has evolved independent of publisher. So they have two versions. IMO that is acceptable.